src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertyAccessorSymbolReferenceFinder.cs

```diff
@@ -34,14 +32,64 @@ protected override bool CanFind(IMethodSymbol symbol)
             return result;
         }
 
-        protected override Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, SymbolFinderOptions options, CancellationToken cancellationToken)
         {
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name);
+            var explicitUsages = await FindDocumentsAsync(project, documents, cancellationToken, symbol.Name).ConfigureAwait(false);
+
+            var property = symbol.AssociatedSymbol as IPropertySymbol;
+            if (property == null || !options.IncludeImplicitAccessorUsages)
+            {
+                return explicitUsages;
+            }
+
+            return explicitUsages.Concat(await FindDocumentsAsync(project, documents, cancellationToken, property.Name).ConfigureAwait(false));
         }
 
-        protected override Task<ImmutableArray<ReferenceLocation>> FindReferencesInDocumentAsync(IMethodSymbol symbol, Document document, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<ReferenceLocation>> FindReferencesInDocumentAsync(IMethodSymbol symbol, Document document, SymbolFinderOptions options, CancellationToken cancellationToken)
         {
-            return FindReferencesInDocumentUsingSymbolNameAsync(symbol, document, cancellationToken);
+            var explicitUsages = await FindReferencesInDocumentUsingSymbolNameAsync(symbol, document, cancellationToken).ConfigureAwait(false);
+
+            var property = symbol.AssociatedSymbol as IPropertySymbol;
+            if (property == null || !options.IncludeImplicitAccessorUsages)
+            {
+                return explicitUsages;
+            }
+
+            // Picks up whatever an IPropertySymbol search would find, e.g. foreach and element access.
+            var propertyReferences = await ReferenceFinders.Property.FindReferencesInDocumentAsync(
+                SymbolAndProjectId.Create(property, document.Project.Id),
+                document,
+                options,
+                cancellationToken).ConfigureAwait(false);
+
+            var results = ImmutableArray.CreateBuilder<ReferenceLocation>();
+            results.AddRange(explicitUsages);
+
+            var isSetterSearch = symbol == property.GetMethod;
```

Comment:
it's a setter search.... if the symbol is the GetMethod?  This is either correct (and needs a comment to explain what's going on), or it's wrong. :)

Reply:
Definitely should be `!=` there!

Reply:
Definitely should be `!=` there! There's a bug running the VSIX which won't show text for the lines, only the count and a blank line per result. I'll double down with tests once I know for sure we're keeping this code.